src/Indice.Features.GovGr/Configuration/GovGrOptions.cs

```diff
@@ -77,4 +79,33 @@ public class BusinessRegistryOptions
         /// <summary>Callers Fiscal Code</summary>
         public string CallersFiscalCode { get; set; }
     }
-}
+
+    /// <summary>Bancapp settings</summary>
+    public class BancappOptions
+    {
+        /// <summary>Represents the environment. Valid options are <em>production</em>, <em>staging</em>, <em>development</em> &amp; <em>mock</em>.Defaults to <b>stage</b>. </summary>
+        public string Environment { get; set; }
```

Comment:
Consider changing this to an Enum of fixed environment names

Reply:
Will do in the next iteration